packages/atomic/src/components/search/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -224,7 +224,9 @@ export class AtomicCategoryFacet
       sortCriteria: this.sortCriteria,
       facetSearch: {numberOfValues: this.numberOfValues},
       basePath: this.basePath
-        ? this.basePath.split(',').map((value) => value.trim())
+        ? this.basePath
+            .split(this.delimitingCharacter)
+            .map((value) => value.trim())
```

Comment:
It's pretty intuitive to me that the prop is split by commas even if that's ont the delimiting character 🤔  I'm curious what Stencil's array props do.

Either way, we should update the prop's description to reflect whichever way we go.